astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -359,13 +360,18 @@ def test_invalid_slices_errors(ignore_matplotlibrc):
 
     with pytest.raises(ValueError) as exc:
         plt.subplot(1, 1, 1, projection=wcs3d)
-    assert exc.value.args[0] == ("WCS has more than 2 pixel dimensions, so "
-                                 "'slices' should be set")
+    assert (
+        exc.value.args[0]
+        == "WCS has more than 2 pixel dimensions, so 'slices' should be set"
+    )
 
     with pytest.raises(ValueError) as exc:
         plt.subplot(1, 1, 1, projection=wcs3d, slices=('x', 'y', 1, 2))
-    assert exc.value.args[0] == ("'slices' should have as many elements as "
-                                 "WCS has pixel dimensions (should be 3)")
+    assert (
+        exc.value.args[0]
+        == "'slices' should have as many elements as "
+        "WCS has pixel dimensions (should be 3)"
+    )
```

Comment:
As above - maybe we really need to just go through all the pytest.raises and make sure they use match=! (I am mentioning this because I find the black formatted code here quite unreadable)

Reply:
Fixed by 5dded7a.